src/components/Modals/DeployModal/DeployToLand/LandAtlas/LandAtlas.tsx

```diff
@@ -238,6 +238,8 @@ export default class LandAtlas extends React.PureComponent<Props, State> {
         <div className="atlas-container">
           <Atlas
             showControls
+            showOperator
+            showTenant
```

Comment:
isn't this missing the `showOwner` too?

Reply:
I thought so, but the [Atlas component](https://github.com/decentraland/ui/blob/447d1ec42febbcee16dc407a62e4010f9825d624/src/components/Atlas/Atlas.tsx#L30) seems to be showing them by default with the default layer that it has.